src/components/UserList/UserList.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import {
+ StyleSheet,
+ Text,
+ View,
+ Navigator,
+ Image,
+ ListView,
+} from 'react-native';
+import Header from './Header';
+import Footer from './Footer';
+import Row from './Row';
+
+const API_KEY = '73b19491b83909c7e07016f4bb4644f9:2:60667290';
```

Comment:
Hm. Could you point out where this API_KEY goes into each native code? We should probably encrypt the key for security purposes.